src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4842,10 +4842,11 @@ namespace N
 
     class C
     {
+        private const FormattableString {|Rename:Formattable|} = $"""";
```

Comment:
>private const FormattableString {|Rename:Formattable|} = $""""; [](start = 8, length = 63)

This doesn't look right. 
1) Why it became a field?
2) It feels like this is not going to compile.

Reply:
You're right, I didn't review the diff well. Will take a look.